configs/components/echam/echam.yaml

```diff
@@ -496,6 +492,46 @@ choose_yr_perp_off:
                                 runctl:
                                         l_orbsvop87: ".false."
 
+choose_with_lco2_emis:
+    True:
+        # Run "emission-driven" echam experiment (in cmip6 context e.g. `esm-piControl` or `esm-hist`) where
+        # atmosheric CO2 concentration is not prescribed but calculated by the model setup.
+        # CO2 emissions need to be provided as forcing file (`carbon_emissions.nc`).
+        add_namelist_changes:
+            namelist.echam:
+                submodelctl:
+                    lco2: true # co2 submodel interacting with jsbach; true for emission-driven; if true, lco2_scenario will be set false during model run
+                radctl:
+                    ico2: 1 # 1 = interactively calculated co2 for emission-driven (lco2=true)
```

Comment:
Not really important here, but I would really like to have some link to the various echam options. +1 to Chris for a good comment!

Reply:
Why not important?

Reply:
those options are all described in the ECHAM6 user documentation, although I have to add the sometimes the document is missing some of the options that were introduced just before ECHAM6's development was stopped.